app/controllers/api/v1/users/friends_controller.rb

```diff
@@ -6,7 +6,15 @@ module Users
       class FriendsController < ApplicationController
         def index
           user = User.find(params[:user_id])
-          render json: FriendsSerializer.new(user.buds)
+          render json: FriendsSerializer.new(user)
+        end
+
+        def update
+          render json: FriendsFacade.accept_friendship(params[:user_id], params[:friend_id])
+        end
+
+        def create
+          render json: FriendsFacade.create_friendship(params[:user_id], params[:friend_email])
         end
```

Comment:
I didn't realize a facade was something you could utilize for exposing endpoints, cool!